### PR TITLE
[admin]list user

### PR DESCRIPTION
**Pull request by deviltruong**

### CODE REVIEW

File: module/Admin/config/module.config.php [hunk 1]
```diff
@@ -25,11 +27,26 @@
...
+                        'action'        => 'index',
+                    ],
+                ],
+            ],
         ],
     ],
     'controllers' => [
         'factories' => [
             Controller\IndexController::class => InvokableFactory::class,
+            Controller\UserController::class => Controller\Factory\UserControllerFactory::class,
```
Comment (ngtrduc) [inline module/Admin/config/module.config.php]: 80 kitu

File: module/Admin/src/Controller/Factory/UserControllerFactory.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+/**
+ * This is the factory for AdminController. Its purpose is to instantiate the
+ * controller.
+ */
+class UserControllerFactory implements FactoryInterface
+{
+	public function __invoke(ContainerInterface $container,
+		$requestedName, array $options = null)
+	{
+		$entityManager = $container->get('doctrine.entitymanager.orm_default');
```
Comment (ngtrduc) [inline module/Admin/src/Controller/Factory/UserControllerFactory.php]: 80 kitu

File: module/Admin/src/Controller/UserController.php [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    public function indexAction(){
+    	return ViewModel();
+    }
+    public function listAction(){
+    	$users = $this->entityManager->getRepository(User::class)->findAll();
+        // Render the view template
+        return new ViewModel([
+            'users' => $users
+            ]);
+
```
Comment (ngtrduc) [inline module/Admin/src/Controller/UserController.php]: thua dong enter

File: module/Admin/src/Form/UserForm.php [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (ngtrduc) [inline module/Admin/src/Form/UserForm.php]: file khong can thi xoa di

File: module/Admin/view/admin/user/list.phtml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    </thead>
+    <tbody>
+        <?php foreach($users as $user): ?>
+            <tr>
+                <td><?php echo $user->getName() ?></td>
+                <td><?php echo $user->getEmail() ?></td>
+                <td><?php echo $user->getPhone() ?></td>
+                <td><?php echo $user->getAddress() ?></td>
+                <td><a class="btn btn-warning" href="#">Show</a>
+                    <form method="POST" action="#" accept-charset="UTF-8" id="form-delete" style="display: inline-block;">
```
Comment (ngtrduc) [inline module/Admin/view/admin/user/list.phtml]: 80 kitu

**Comment by trinhductoan** [comment]
OK

**Pull request merged**